src/man/sssd.conf.5.xml

```diff
@@ -2964,6 +2966,16 @@ min_id = 10000
 max_id = 20000
 enumerate = False
 </programlisting>
+	</para>
+	<para>
+            2. The following example shows configuration in IPA AD trust. AD having
+            having child domain. Suppose IPA domain(ipa.com) has trust with AD
```

Comment:
I'd add an space between "domain"and the "(", like: "Suppose IPA domain (ipa.com) ..."